src/Core/Prefab/GlobeView.js

```diff
@@ -346,4 +623,147 @@ GlobeView.prototype.changeRenderState = function changeRenderState(newRenderStat
     this._renderState = newRenderState;
 };
 
+/**
+ * This function adds an imagery layer to the scene using a JSON file. The layer id must be unique. The protocol rules wich parameters are then needed for the function.
+ * @param {Layer} layer.
+ * @return     {layer}  The Layer.
+ */
+
+GlobeView.prototype.addImageryLayerFromJSON = function addImageryLayerFromJSON(url) {
+    return Fetcher.json(url).then(result => this.addImageryLayer(result));
+};
+
+/**
+ * This function adds an imagery layer to the scene using an array of JSON files. The layer id must be unique. The protocol rules wich parameters are then needed for the function.
+ * @param {Layers} array - An array of JSON files.
+ * @return     {layer}  The Layers.
+ */
+GlobeView.prototype.addImageryLayersFromJSONArray = function addImageryLayersFromJSONArray(urls) {
+    const proms = [];
+
+    for (const url of urls) {
+        proms.push(Fetcher.json(url).then(layer => this.addImageryLayer(layer)));
+    }
+
+    return Promise.all(proms);
+};
+
+/**
+ * Add an elevation layer to the map using a JSON file.
+ * Elevations layers are used to build the terrain.
+ * Only one elevation layer is used, so if multiple layers cover the same area, the one
+ * with best resolution is used (or the first one is resolution are identical).
+ * The layer id must be unique amongst all layers already inserted.
+ * The protocol rules which parameters are then needed for the function.
+ * @param {Layers} array - An array of JSON files.
+* @return     {layer}  The Layers.
+ */
+GlobeView.prototype.addElevationLayersFromJSON = function addElevationLayersFromJSON(url) {
+    return Fetcher.json(url).then(result => this.addElevationLayer(result));
+};
+
+/**
+ * Add an elevation layer to the map using an array of JSON files.
+ * Elevations layers are used to build the terrain.
+ * Only one elevation layer is used, so if multiple layers cover the same area, the one
+ * with best resolution is used (or the first one is resolution are identical).
+ * The layer id must be unique amongst all layers already inserted.
+ * The protocol rules which parameters are then needed for the function.
+ * @param {Layer} layer.
+ * @return     {layer}  The Layers.
+ */
+GlobeView.prototype.addElevationLayersFromJSONArray = function addElevationLayersFromJSONArray(urls) {
```

Comment:
The functions  `add*FromJSONArray` don't belong to itowns IMHO.

And I feel the same regarding the `add*Layer` actually.
iTowns provides 1 way to add a layer (`view.addLayer`) which cover all our needs - the specific functions should live in the application using itowns.